.github/workflows/pull-request.yml

```diff
@@ -1,6 +1,8 @@
 name: Pull Request
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, edited]
 permissions: read-all
 jobs:
   quality:
```

Comment:
I believe this is where @HarryLinux suggested we add `uses: actions/checkout@v3` to checkout the repo before we run the Quality workflow.